src/views/TechnicalAssistance/TrbAdminTeamHome.tsx

```diff
@@ -0,0 +1,648 @@
+                      {column.render('Header')}
+                    </div>
+                    <div className="position-relative width-205 margin-left-05">
+                      {getHeaderSortIcon(column)}
+                    </div>
+                  </Button>
+                </th>
+              ))}
+            </tr>
+          ))}
+        </thead>
+        <tbody {...getTableBodyProps()}>
+          {page.map((row, rowIdx) => {
+            return (
+              <tr {...row.getRowProps()}>
+                {row.cells.map((cell, index) => {
+                  return (
+                    <td
+                      data-testid={`trb-existing-cell-${rowIdx}-${index}`}
+                      {...cell.getCellProps()}
+                    >
+                      {cell.render('Cell')}
+                    </td>
+                  );
+                })}
+              </tr>
+            );
+          })}
+        </tbody>
+      </Table>
+
+      {rows.length === 0 && (
+        <div className="padding-x-2 padding-bottom-1 border-bottom-1px margin-top-neg-105 line-height-body-5">
+          {t(`adminTeamHome.existingRequests.noRequests.${activeTable}`)}
+        </div>
+      )}
+
+      {rows.length > 0 && (
+        <>
+          <div className="grid-row grid-gap grid-gap-lg">
+            <TablePagination
+              gotoPage={gotoPage}
+              previousPage={previousPage}
+              nextPage={nextPage}
+              canNextPage={canNextPage}
+              pageIndex={state.pageIndex}
+              pageOptions={pageOptions}
+              canPreviousPage={canPreviousPage}
+              pageCount={pageCount}
+              pageSize={state.pageSize}
+              setPageSize={setPageSize}
+              page={[]}
+              className="desktop:grid-col-fill desktop:padding-bottom-0 desktop:margin-bottom-0"
+            />
+            <TablePageSize
+              className="desktop:grid-col-auto"
+              pageSize={state.pageSize}
+              setPageSize={setPageSize}
+            />
+          </div>
+
+          <div
+            className="usa-sr-only usa-table__announcement-region"
+            aria-live="polite"
+          >
+            {currentTableSortDescription(headerGroups[0])}
+          </div>
+        </>
+      )}
+    </div>
+  );
+}
+
+function TrbAdminTeamHome() {
+  const { t } = useTranslation('technicalAssistance');
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const { loading, error, data } = useQuery<GetTrbAdminTeamHome>(
+    GetTrbAdminTeamHomeQuery
+  );
+
+  // todo Refer to EASI-2725 on when backend will handle this filtering
+  const trbRequests = data?.trbRequests.filter(
+    r => r.form.submittedAt !== null
+  );
+
+  return (
+    <GridContainer className="width-full">
+      <PageHeading className="margin-bottom-1">{t('heading')}</PageHeading>
+      <div className="font-body-lg">{t('adminTeamHome.description')}</div>
```

Comment:
This needs a bit more vertical spacing and should have the `text-light` class to match the Figma.

Also, text should be wrapped in `<p>` instead of `<div>`.